README.md

```diff
@@ -16,28 +16,30 @@ Features:
 - Endpoints can be mocked
   - Write to it to instruct it to send back messages the same way a real device would
   - Allows separating TTY message logic from actual devices for rapid prototyping
+car
+- Communication between server and client is done via the [WebSocket Protocol][https://datatracker.ietf.org/doc/html/rfc6455]
+  - Allows any external clients to subscribe to serial-keel server
 
-## Quickstart
-
-If you have installed serial keel (see [the install step](#running-a-server) below), please use `serial-keel help` to explore what's available.
-
-### Running a server
+## Running a server
 Start the server. Choose one of:
 
-1. `cargo r`, or `cargo r --release`, or, `cargo r --features mocks-share-endpoints --release` (see [cargo features](#cargo-features))
-3. Install, e.g.: `cargo install --path serial-keel --features mocks-share-endpoints`, then run it: `serial-keel`
-2. Precompiled: `./bin/serial-keel` (TODO: Update this bin)
+* `cargo r`, or `cargo r --release`, or, `cargo r --features mocks-share-endpoints --release` (see [cargo features](#cargo-features))
+* Install, e.g.: `cargo install --path . --features mocks-share-endpoints`, then run it: `serial-keel`
+* Precompiled: `./bin/serial-keel` (TODO: Update this bin)
+
+If you have installed serial keel, please use `serial-keel help` to explore what's available.
 
 Use the environment variable `RUST_LOG` to control log verbosity, e.g. `export RUST_LOG=warn` or `export RUST_LOG=debug`.
 
 ### Using a configuration file
 
+By default all all serial ports will be opened to prevent this a configuration file is needed.
 Run `serial-keel examples config` to see an example configuration file.
 You can store this as `my-config.ron` to get started.
 
 ## How it works
 
-### Concepts
+### Entities
```

Comment:
As this section only gives an overview of which Entities are involved but not really a lot about how they communicate and states, I felt this would be a more suitable name for the section, but make there is more coming in the future :)